server/controllers/user.controller.js

```diff
@@ -50,12 +50,14 @@ class UserController {
       }
     };
 
+    const body = `<p>Thanks for joining and we hope you read stories that change your life forever!</p>
+    Warm regards.`;
+
     const mailOption = {
       email,
       subject: 'Welcome to Author\'s Haven',
-      message: `<h3>Hi ${firstname},
-      <p>Thanks for joining and we hope you read stories that change your life forever!</p>
-      Warm regards.`,
+      message: mailTemplate(`<h3>Hello ${firstname} ${lastname},</h3>`, body)
```

Comment:
Hi, @Chiazokam I think we can do without the `<h3>` tag here. It was not used in the `login` mail.

Reply:
Alright....will do this

Reply:
Done